### DISCUSSION TITLE
Add Data Flows to Directus 🌊

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#13422] [Archive]
#12522 was the last point hanging for 9.11.0 (from a 100 resolved - congrat), but finally did not make it.
What is it's current status now? and what can we reasonably expect in the near future for this promising new feature?

**Comment by rijkvanzanten** [comment]
> was the last point hanging for 9.11.0 (from a 100 resolved - congrat)

It was mostly a "Lets see if we can merge it before 9.11" more than a "This has to make it before 9.11" 😉 

> What is it's current status now? and what can we reasonably expect in the near future for this promising new feature?

It's very close to being done, just wrapping up some of the last points / bugs / tweaks now. I would highly recommend reading through the (long) thread to get fully up to speed with the current status and everything that's included 👍🏻

**Comment by Pachat** [comment]
Its now there :) Flows are great ! Thanks to the developers. I'll sponsor :)
